### ISSUE TITLE
stuck on Loading dependency graph, done.

### ISSUE THREAD

**Issue by Hitsaa** [facebook/react-native#27616]
React Native version:

System:
    OS: Windows 10 10.0.18362
    CPU: (4) x64 Intel(R) Core(TM) i5-6200U CPU @ 2.30GHz
    Memory: 1.06 GB / 7.90 GB
  Binaries:
    Node: 10.16.3 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.19.1 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
    npm: 6.9.0 - C:\Program Files\nodejs\npm.CMD
    Watchman: 4.9.4 - C:\Users\shekh\AppData\Local\watchman\windows\bin\watchman.EXE
  SDKs:
    Android SDK:
      API Levels: 23, 25, 27, 28, 29
      Build Tools: 28.0.3, 29.0.2
      System Images: android-22 | Google APIs Intel x86 Atom
  IDEs:
    Android Studio: Version  3.5.0.0 AI-191.8026.42.35.6010548
  npmPackages:
    react: 16.9.0 => 16.9.0
    react-native: 0.61.5 => 0.61.5

## Steps To Reproduce

1.react-native run-android

Describe what you expected to happen:
it should install app in emulator or device but not going further to as shown in screen shot.
![Screenshot (1)](https://user-images.githubusercontent.com/33285810/71466233-f4203680-27e4-11ea-8728-91acf5e6e904.png)

I applied following solutions and none of them worked.
Sol. 1. Clearing package manager
Steps:
1. watchman watch-del-all
2. rm -r node_modules/ 
3. npm cache clean 
4. npm install 
5. rm -fr $TMPDIR/metro*
6. npm start -- --reset-cache

Sol. 2. Restart PC
Sol. 3. Update react-native version by react-native upgrade.
Sol. 4. react-native start and then react-native run-android
Sol. 5.  npm start and then react-native run-android
Sol. 6. running react-native run-android from command prompt
sol. 7. running react-native start from command prompt. then open another command prompt 
          and run react-native run-android
sol. 8 yarn start
sol. 9 yarn start and then yarn react-native run-android

**Comment by builtbyproxy** [comment]
Hey @Hitsaa Do you have an emulator running by the time you hit that `Loading Dependency Graph`? 

When you run `react-native run-android` the following should happen: 
- Your android emulator will open
- Your app will be installed
- the `Loading Dependency Graph` terminal will load

From there if nothing is happening try refreshing the app in your emulator. To do this either double tap `r` OR hit `CMD+M` to open the debugging menu and then select Reload

-------

Assuming your emulator did not open you will need to either: 
1. Plug in an android device
2. Create an android emulator. 

`1.` is obvious (ensure you have usb debugging enabled, this can be googled if you are unsure how). However, 2 is a little more trick, to create an emulator do the following: 

1. Open your android studio
2. In the top right select this button: 
![image](https://user-images.githubusercontent.com/16897508/71566974-8c7d2a80-2b07-11ea-9db4-dfc7f9038ebf.png)
3. From there a screen similar to this one will popup: 
![image](https://user-images.githubusercontent.com/16897508/71567002-b6365180-2b07-11ea-9791-f479edafac63.png) If you have an emulator already click the green play button to start it, then re-run `react-native run-android`. However if you don't click `Create Virtual Device` in the bottom left
4. Assuming you clicked `Create Virtual Device` you'll be greeted with a select hardware screen. I like to pick some form of Pixel for my emulators, follow the screen through and download whatever SDK you'd like (Pie, Oreo, Nougat). Give it a name and click finish. 
5. Click the green play on your new android emulator and re-run `react-native run-android`

Once you have an emulator up follow my original message of refreshing your white screen if your app installs. If your app doesn't install then let me know here 😃 We still have plenty of options to try 👍

**Comment by Hitsaa** [comment]
@builtbyproxy Thank you for your reply. Your steps were correct and all these steps I had already followed.
Actually I figured out problem.
The Problem was in the react dependency "react-native-vector-icons". This dependency was creating problem in the loading packages and installing the app in my emulator and real mobile device.
As soon as I removed the dependency app was installing in the emulator and device. But of-course as my code was using "react-native-vector-icons" dependency, I had to install dependency in some different way. So I figured out the way as given in https://github.com/oblador/react-native-vector-icons/issues/1035#issuecomment-509836041  and further my app was working fine in the emulator as well as in mobile device.
